packages/ui/components/pages/Fuse/FusePoolPage/BorrowList.tsx

```diff
@@ -43,6 +43,7 @@ export const BorrowList = ({ assets, borrowBalanceFiat, comptrollerAddress }: Bo
   );
   const unBorrowableAssets = assets.filter((asset) => asset.isBorrowPaused);
 
+  // eslint-disable-next-line no-console
```

Comment:
no console in the next line, so could be removed.
```suggestion
  // eslint-disable-next-line no-console
```